Code/Framework/AzCore/AzCore/Debug/Trace.cpp

```diff
@@ -224,6 +224,8 @@ namespace AZ
 
     void Debug::Trace::Terminate(int exitCode)
     {
+        AZ_TracePrintf("Exit", "Called Terminate() with exit code: 0x%x", exitCode);
+        AZ::Debug::Trace::PrintCallstack("Exit");
```

Comment:
Is the Terminate callstack useful?

Terminate is a more hard coded call, so the callstack is easily able to be looked by the searching the calls within the code.

Outputting the callstack on an exception is more useful, since the stack would be different in that case.